Source/WebCore/page/FocusController.cpp

```diff
@@ -250,6 +290,8 @@ Element* FocusNavigationScope::owner() const
     ASSERT(m_treeScopeRootNode);
     if (is<ShadowRoot>(*m_treeScopeRootNode))
         return downcast<ShadowRoot>(*m_treeScopeRootNode).host();
+    if (isOpenPopoverWithInvoker(m_treeScopeRootNode.get()))
+        return dynamicDowncast<HTMLElement>(m_treeScopeRootNode.get())->popoverData()->invoker();
```

Comment:
I don't think this will work. Compiler will turn dynamicDowncast into a regular downcast since nullptr deference results in undefined behavior. We should release assert that dynamicDowncast<HTMLElement>(m_treeScopeRootNode.get()) is not null instead.